modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/DistributedSqlConfiguration.java

```diff
@@ -58,6 +63,9 @@
     private final SimpleDistributedProperty<HashSet<String>> disabledSqlFuncs
         = new SimpleDistributedProperty<>("sql.disabledFunctions");
 
+    /** Value of cluster time zone. */
+    private final SimpleDistributedProperty<TimeZone> timeZone = new SimpleDistributedProperty<>("sqlTimeZone");
```

Comment:
it's better to name property `sql.timeZone` since there is already `sql.disabledFunctions `

Reply:
Thanks for your attention!